src/db/base.db.gateway.ts

```diff
@@ -0,0 +1,18 @@
+import { EntityManager, Repository } from 'typeorm';
+import { ObjectLiteral } from 'typeorm/common/ObjectLiteral';
+
+export abstract class BaseDbGateway<Entity extends ObjectLiteral> {
+
+    constructor(
+        protected readonly repository: Repository<Entity>
+    ) {}
+
+    /**
+     * Helper function to support executing database actions transactionally. Database actions must use the provided EntityManager in order to be
+     * executed transactionally.
+     */
+    protected async runInTransaction<T>(fun: (entityManager: EntityManager) => Promise<T>): Promise<T> {
+        return this.repository.manager.transaction(fun);
```

Comment:
Thanks @jsaur this is the same issue I was seeing too. I did have it working locally (I thought), but it may have been a stale local cache issue since it's not working now.